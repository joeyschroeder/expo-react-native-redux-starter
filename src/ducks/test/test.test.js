import { DEFAULT_STATE, resetTestData, testDuck, updateTestData } from './test.duck';

describe('testDuck', () => {
  describe('updateTestData', () => {
    it('should add one to data', () => {
      const result = testDuck(DEFAULT_STATE, updateTestData());
      expect(result).toMatchSnapshot();
    });
  });

  describe('resetTestData', () => {
    it('should reset data', () => {
      const result = testDuck(DEFAULT_STATE, resetTestData());
      expect(result).toMatchSnapshot();
    });
  });
});
