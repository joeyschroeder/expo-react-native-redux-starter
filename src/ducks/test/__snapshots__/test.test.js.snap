// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testDuck resetTestData should reset data 1`] = `
Object {
  "additionalState": "test",
  "data": 0,
}
`;

exports[`testDuck updateTestData should add one to data 1`] = `
Object {
  "additionalState": "test",
  "data": 1,
}
`;
